pages/CreatingAnExploration.html

```diff
@@ -12,13 +12,13 @@ <h2>
     do so, click the "Create" button in the top navbar.
   </p>
 
-  <img src="static/images/libraryCreateButton.png" style="width: 100%;">
+  <img src="static/images/libraryCreateButton.png" style="width: 100%;" alt="A top-navigation bar having various options such as 'library', 'about' and 'donate' along with a 'create' button circled in yellow. ">
```

Comment:
Try to describe the image so that someone hearing the description can recreate it in their mind. In this case it is probably better to list all the options explicitly and not just have a "such as".